src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/FrameComponent.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Applications.ComponentBased.Common
+{
+    /// <summary>
+    /// The class for showing UI module
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public abstract class FrameComponent : BaseComponent
+    {
+        /// <summary>
+        /// Enumeration for display status.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public enum DisplayStatus
+        {
+            /// <summary>
+            /// The display status is on
+            /// </summary>
+            On = Interop.CBApplication.NativeDisplayStatus.On,
+
+            /// <summary>
+            /// The display status is off
+            /// </summary>
+            Off = Interop.CBApplication.NativeDisplayStatus.Off
+        }
+
+        /// <summary>
+        /// Gets the display status of a component.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when component type is already added to the component.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public DisplayStatus CurrentDisplayStatus
+        {
+            get
+            {
+                Interop.CBApplication.NativeDisplayStatus status;
+                Interop.CBApplication.ErrorCode err = Interop.CBApplication.BaseFrameGetDisplayStatus(Handle, out status);
+                if (err != Interop.CBApplication.ErrorCode.None)
+                    throw new InvalidOperationException("Fail to get display status : err(" + err + ")");
+
+                return (DisplayStatus)status;
+            }
+        }
+
+        /// <summary>
+        /// Gets the frame component's window.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public IWindow Window { get; internal set; }
+
+        /// <summary>
+        /// Overrides this method if want to handle behavior when the component is launched.
+        /// </summary>
+        /// <returns>Window object to use</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public virtual IWindow OnCreate()
```

Comment:
It should be abstract
```suggestion
        public abstract IWindow OnCreate();
```

Reply:
Done